relay-event-schema/src/protocol/span.rs

```diff
@@ -175,6 +175,16 @@ impl Getter for Span {
 #[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
 #[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
 pub struct SpanData {
+    /// Mobile app start variant.
+    ///
+    /// Can be either "cold" or "warm".
+    #[metastructure(field = "app_start_type")] // TODO: no dot?
+    pub app_start_type: Annotated<Value>,
```

Comment:
All these fields are weakly typed, because they can theoretically contain any data, even if we have conventions around them.